draft-irtf-cfrg-frost.md

```diff
@@ -514,49 +530,55 @@ This section describes the subroutine for creating the group commitment
 from a commitment list.
 
 ~~~
-  Inputs:
-  - commitment_list =
-     [(i, hiding_nonce_commitment_i, binding_nonce_commitment_i), ...], a list
-    of commitments issued by each participant, where each element in the list
-    indicates a NonZeroScalar identifier i and two commitment Element values
-    (hiding_nonce_commitment_i, binding_nonce_commitment_i). This list MUST be
-    sorted in ascending order by identifier.
-  - binding_factor_list = [(i, binding_factor), ...],
-    a list of (NonZeroScalar, Scalar) tuples representing the binding factor Scalar
-    for the given identifier.
-
-  Outputs:
-  - group_commitment, an Element.
-
-  def compute_group_commitment(commitment_list, binding_factor_list):
-    group_commitment = G.Identity()
-    for (identifier, hiding_nonce_commitment, binding_nonce_commitment) in commitment_list:
-      binding_factor = binding_factor_for_participant(binding_factor_list, identifier)
-      group_commitment = group_commitment +
-        hiding_nonce_commitment + G.ScalarMult(binding_nonce_commitment, binding_factor)
-    return group_commitment
+Inputs:
+- commitment_list = [(i, hiding_nonce_commitment_i,
+  binding_nonce_commitment_i), ...], a list of commitments issued by
+  each participant, where each element in the list indicates a
+  NonZeroScalar identifier i and two commitment Element values
+  (hiding_nonce_commitment_i, binding_nonce_commitment_i). This list
+  MUST be sorted in ascending order by identifier.
+- binding_factor_list = [(i, binding_factor), ...],
+  a list of (NonZeroScalar, Scalar) tuples representing the binding
+  factor Scalar for the given identifier.
+
+Outputs:
+- group_commitment, an Element.
+
+def compute_group_commitment(commitment_list, binding_factor_list):
+  group_commitment = G.Identity()
+  for (identifier, hiding_nonce_commitment,
+       binding_nonce_commitment) in commitment_list:
+    binding_factor = binding_factor_for_participant(
+        binding_factor_list, identifier)
+    group_commitment = (
+        group_commitment +
+        hiding_nonce_commitment +
+        G.ScalarMult(
+            binding_nonce_commitment,
+            binding_factor))
```

Comment:
```suggestion
    binding_nonce = G.ScalarMult(
            binding_nonce_commitment,
            binding_factor)
    group_commitment =
        group_commitment +
        hiding_nonce_commitment +
        binding_nonce
```

Reply:
Good call, I went with a similar approach in b376ec0 but keeping the Python syntax